pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -307,7 +312,7 @@ pub mod pallet {
 			let avatar = Avatar { season: active_season_id, dna };
 			let avatar_id = T::Hashing::hash_of(&avatar);
 
-			let mut avatars = Players::<T>::take(&player);
+			let mut avatars = Players::<T>::get(&player);
```

Comment:
`take` actually removes entries from storage so it's safer to use `get` (technically the extrinsic can't fail at this point but it's good practice).

Reply:
Good point. Changed!